src/wpanctl/tool-cmd-config-gateway.c

```diff
@@ -201,24 +204,17 @@ int tool_cmd_config_gateway(int argc, char* argv[])
 				}
 			}
 
-			fprintf(stderr, "Using prefix \"%s\"\n", prefix);
+			inet_ntop(AF_INET6, (const void *)&prefix_bytes, address_string, sizeof(address_string));
+
+			fprintf(stderr, "Using prefix \"%s\"\n", address_string);
 
-			uint8_t *addr = prefix_bytes;
-			dbus_message_append_args(
-			    message,
-			    DBUS_TYPE_ARRAY, DBUS_TYPE_BYTE, &addr, 8,
-			    DBUS_TYPE_INVALID
-			);
-		} else {
-			dbus_message_append_args(
-			    message,
-			    DBUS_TYPE_ARRAY, DBUS_TYPE_BYTE, "", 0,
-			    DBUS_TYPE_INVALID
-			);
 		}
 
+		addr = prefix_bytes;
+
 		dbus_message_append_args(
 		    message,
+			DBUS_TYPE_ARRAY, DBUS_TYPE_BYTE, &addr, 16,
```

Comment:
nit: I think this is tab vs space, `DBUS_TYPE_ARRAY` does not look aligned with the rest.